The benchmark for the `trace` operator always fail.

### Describe the bug

The `test_trace` test case for benchmark always fail.

```
FAILED benchmark/test_trace.py::test_trace - Failed: trace(): argument 'input' (position 1) must be Tensor, not NoneType
```

--- a/benchmark/test_trace.py
+++ b/benchmark/test_trace.py
@@ -11,7 +11,6 @@ def _input_fn(shape, dtype, device):
     yield inp,
 
 
-@pytest.mark.skip(reason="Test case failure: issue #2663")
 @pytest.mark.trace
 def test_trace():
     if flag_gems.vendor_name == "mthreads":

